packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -88,7 +86,7 @@ class RenderingEngine implements IRenderingEngine {
    * @param uid - Unique identifier for RenderingEngine
    */
   constructor(id?: string) {
-    this.id = id ? id : uuidv4();
+    this.id = id ? id : crypto.randomUUID();
```

Comment:
Not sure why you are using this, since uuidv4 uses crypto internally

Reply:
Trying to reduce imports mostly, since it is now native to browser/nodejs environments, and produces the same results.

Reply:
I think it is fine honestly